### DISCUSSION TITLE
Order Relationships

### DISCUSSION THREAD

**Discussion by ddcollins** [directus/directus#3246] [Feature Requests]
I have a data model called "Course" which has a "many to many" relationship defined through "steps" that are videos. The videos must be presented in a specific order, but currently there is no way to define the order through the relationship.

This would be nice to have for M2A as well!

![Image 2020-11-30 at 2 33 12 PM](https://user-images.githubusercontent.com/38921476/100661710-7bd9c380-3319-11eb-803d-94c5342d810c.png)

Contentful handles this well:

![Image 2020-11-30 at 2 37 47 PM](https://user-images.githubusercontent.com/38921476/100661791-a1ff6380-3319-11eb-8666-451cf605da74.png)

**Comment by rijkvanzanten** [comment]
@ddcollins The interface actually has this built in, I just have to show the option in the interface settings 😁 

RC20!
